src/store/conversationsStore.js

```diff
@@ -317,10 +318,12 @@ const actions = {
 	 * Delete a conversation from the store.
 	 *
 	 * @param {object} context default store context;
-	 * @param {object} token the token of the conversation to be deleted;
+	 * @param {string} token the token of the conversation to be deleted;
 	 */
 	deleteConversation(context, token) {
 		// FIXME: rename to deleteConversationsFromStore or a better name
+		const chatExtrasStore = useChatExtrasStore()
+		chatExtrasStore.purgeChatExtras(token)
```

Comment:
Same for `clearMessagesHistory()` ?

Reply:
That would be an unexpected behaviour, if you click on 'Delete chat messages' and:
- Clear your input
- Remove absence status